xbmc/filesystem/ZeroconfDirectory.cpp

```diff
@@ -67,8 +65,6 @@ namespace
       fr_protocol = "smb";
     else if(fcr_service_type == "_ftp._tcp.")
       fr_protocol = "ftp";
-    else if(fcr_service_type == "_htsp._tcp.")
-      fr_protocol = "htsp";
```

Comment:
should still work https://github.com/opdenkamp/xbmc/commit/a6fb85d54055b1aece2acc860a97a5c4faf6ed96#diff-f5e83012c02818d2afdba8cde071a54dR1769